scripts/build.sh

```diff
@@ -235,6 +235,12 @@ then
     fi
 fi
 
+# TODO: Add check for -d (but not df).
+if [ -z "$AMPLITUDE_API_KEY" ]
```

Comment:
this should go on line 214 above.
It can check for non-force release like so: `if [[ FORCE_RELEASE -eq 0 ]]` see examples in the if-block near line 214 above.

also, in the same way that you've used `$AMPLITUDE_API_KEY` to access the env var here, you should inject it in the `LDFLAGS_ARRAY` array above. This can be set on the root.go file for now. See `version` variable as an example throughout the codebase.

Additionally, we need the kelp code to also do a check where if it is in release mode and does not have an API key it should crash the bot. this can be done in `runTradeCmd`